### PR TITLE
Updated and added code comments v0.1

### PR DESCRIPTION
**Pull request by Pheosics**
First pass at adding comments to various parts of the code, and
expanding/updating already existing comments

### CODE REVIEW

**Comment by Pheosics** [comment]
@drsteve Starting to add comments to the code where needed and cleaning up various formatting issues

**Comment by drsteve** [comment]
All looking good so far... let me know if/when you want me to do anything about this.

**Comment by Pheosics** [comment]
I've got some more code comments coming in soon, figured I would just make the draft pull request now so I can keep track of the changes a little easier

**Comment by Pheosics** [comment]
@drsteve I think these comments are ready to be merged in, I doubt I will have the time to write many more in the next several weeks/months

File: src/ModRamScb.f90 [hunk 1]
```diff
@@ -246,14 +252,6 @@ SUBROUTINE computehI(iter)
        call system_clock(time1,clock_rate,clock_max)
        starttime=time1/real(clock_rate,dp)
 
-       ! If using SWMF without SCB calculation just take directly from grids
```
Comment (drsteve) [inline src/ModRamScb.f90:249]: Question: Why was this removed here? It's obviously not comment-related...

Comment (Pheosics) [reply src/ModRamScb.f90:249]: It probably shouldn't have been removed in this pull request, but it wasn't actually possible to satisfy that particular if statement.
